modules/persistent-storage-csi-vsphere-install-issues.adoc

```diff
@@ -2,23 +2,37 @@
 //
 // persistent-storage-csi-vsphere.adoc
 //
-
+:_content-type: PROCEDURE
 [id="persistent-storage-csi-vsphere-install-issues_{context}"]
 = Removing a third-party vSphere CSI Operator Driver
 
-{product-title} 4.10 includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat. If you have installed a vSphere CSI driver provided by the community or another vendor, upgrades will be prevented in a future release of {product-title}.
+{product-title} {product-version} includes a built-in version of the vSphere Container Storage Interface (CSI) Operator Driver that is supported by Red Hat.
+
+If you have installed a vSphere CSI driver provided by the community or another vendor, which is considered a third-party vSphere CSI driver, and you continue with upgrading to the next subsequent major version of {product-title}, the `oc` CLI prompts you with the following message:
+
+[source,terminal]
+----
+VSphereCSIDriverOperatorCRUpgradeable: VMwareVSphereControllerUpgradeable:
+found existing unsupported csi.vsphere.vmware.com driver
+----
+
+The previous message informs you that Red Hat does not support the third-party vSphere CSI driver during an {product-title} upgrade operation. You can choose to ignore this message and continue with the upgrade operation.
 
-If you remove the third-party vSphere CSI driver, you do not need to delete associated persistent volume (PV) objects, and no data loss should occur.
+The instructions outlined in the procedure demonstrate an example of how to uninstall a third-party vSphere CSI Driver. Consult the vendor's or community provider's uninstall guide for more detailed instructions on removing the driver and its components.
```

Comment:
```suggestion
The instructions outlined in the procedure show an example of how to uninstall a third-party vSphere CSI Driver. Consult the vendor's or community provider's uninstall guide for more detailed instructions on removing the driver and its components.
```
Either one of these options is a suggestion to simplify this sentence.
```suggestion
The instructions outlined in the procedure show how to uninstall a third-party vSphere CSI Driver. Consult the vendor's or community provider's uninstall guide for more detailed instructions on removing the driver and its components.
```

Reply:
Option 2. :heavy_check_mark: